pontoon/batch/actions.py

```diff
@@ -69,6 +70,16 @@ def approve_translations(form, user, translations, locale):
         translations,
         locale,
     )
+
+    # Log actions
```

Comment:
Did you want to make this comment and the one below more specific, to make them in line with the 3rd and the 4th one, e.g. "Log approving actions"?